runtime/neumann/src/lib.rs

```diff
@@ -317,6 +318,7 @@ impl frame_system::Config for Runtime {
 	type SS58Prefix = SS58Prefix;
 	/// The action to take on a Runtime Upgrade
 	type OnSetCode = cumulus_pallet_parachain_system::ParachainSetCode<Self>;
+	type MaxConsumers = frame_support::traits::ConstU32<16>;
```

Comment:
what does this do?

Reply:
/// The maximum number of consumers allowed on a single account.

where the number of consumers on an account:

/// The number of other modules that currently depend on this account's existence. The account
/// cannot be reaped until this is zero.